package.json

```diff
@@ -50,7 +50,8 @@
 		"katex": "^0.16.9",
 		"marked": "^9.1.0",
 		"svelte-french-toast": "^1.2.0",
+		"svelte-typewriter": "^3.2.3",
```

Comment:
We might want to use framework agnostic libraries like `https://www.npmjs.com/package/typewriter-effect` here